spec/latest/core/index.html

```diff
@@ -314,6 +314,11 @@ <h3>Adding Affordances to Representations</h3>
       <code>comments</code> property in the following example is intended
       to be dereferenced.</p>
 
+    <p class="note">It is recommended to dereference resources that are
+      within API's domain. This may prevent possible issues with cross-site
+      scripting or obtaining meaningless resources. Still, there is no formal
```

Comment:
I see what you did there but the word `meaningless` is a bit strong. 

```suggestion
      scripting or obtaining non-information resources. Still, there is no formal
```

Is that correct use of the term `non-information resource`? @asbjornu

Reply:
For the record - I've replaced it as suggested, but I also have some doubts

Reply:
I'm not sure what either "meaningless resources" or "non-information resources" is supposed to mean. Could you please elaborate?

Reply:
Like I say, I'm not sure if this is correct use of the term non-information resource.

I understand this part to say "resources which have not meaning to the client". Or "resources which the client is unable to interpret".

Maybe we should just make this part more verbose but unambiguous?

Reply:
I agree - I'll try to update it ASAP.

Reply:
I've rephrased it so it's more verbose now.